concepticondata/references/references.bib

```diff
@@ -1,3 +1,11 @@
+@Misc{Tjuka2022,
```

Comment:
We quote our blog posts now as a journal. So you should write @article instead.

Like here:

```
@Article{Tjuka2020TBLOG1,
  author    = {Tjuka, Annika},
  title     = {Adding concept lists to Concepticon: A guide for beginners},
  journal   = {Computer-Assisted Language Comparison in Practice},
  year      = {2020},
  number    = {1},
  url       = {https://calc.hypotheses.org/2225},
  volume    = {3},
}
```

This year is volume 5, and page range is the range of the pages in your PDF.

Reply:
So I'd recommend to not quote Zenodo, which is referenced in the blogpost, but rather the article/blog post referencing this list. The URL / DOI can still be added from Zenodo.